python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
+    num = 0
     for letter in text:
         if letter not in vowels:
-            print("consonant", letter)
+           num += text.lower().count(letter)
```

Comment:
as written, this will increment num by the total number of times that letter is in the text for each step through the string. However, if there are repeated letters, those letters will be double counted. I would recommend that you loop through a list of consonants (as in the list of vowels) and use this logic. Otherwise, you could increment num by 1 using this logic, but then you would still have to address characters that are neither vowels nor consonants.